pkg/webhooks/webhooks.go

```diff
@@ -88,16 +92,19 @@ func Start(ctx context.Context, cfg *rest.Config, ctors ...knativeinjection.Cont
 	ctx, startInformers := knativeinjection.EnableInjectionOrDie(ctx, cfg)
 	logger := logging.NewLogger(ctx, component)
 	ctx = knativelogging.WithLogger(ctx, logger)
-
 	cmw := sharedmain.SetupConfigMapWatchOrDie(ctx, knativelogging.FromContext(ctx))
 	controllers, webhooks := sharedmain.ControllersAndWebhooksFromCtors(ctx, cmw, ctors...)
 
 	// Many of the webhooks rely on configuration, e.g. configurable defaults, feature flags.
 	// So make sure that we have synchronized our configuration state before launching the
 	// webhooks, so that things are properly initialized.
 	logger.Info("Starting configuration manager...")
+	log := logctrl.FromContext(ctx)
```

Comment:
Can we drop this and just leave the code the way it was? Could just avoid making any changes here at all since there's no need to use the controller-runtime logger here